.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+### Motivation
```

Comment:
@normanmaurer You're not a fan of hashtags? :)

Honestly, the basic formatting used by this template wouldn't be that ugly with pure text-based tools. The only ugly bit would be if the user added a bunch of markdown within the comments.